app/views/mobile_app/sensors/sponsor.json.jbuilder

```diff
@@ -1 +1,2 @@
 json.extract! @sponsor, :id, :name, :description, :created_at
+json.logo_url image_url(@sponsor.logo_source || 'coredump.png')
```

Comment:
Does this mean that the logo will always be set to `coredump.png` if unset? In that case, it should probably be `null` instead.

Reply:
I more thought of a fallback avatar. Looks nicer and makes the handling in the client easier.

Reply:
Yeah, but it doesn't really make sense to write "Hey, this sponsor is sponsored by HSR" and then put our own logo next to it :slightly_smiling_face: I'd return `null` instead, then the client can decide what to show.

Reply:
See https://github.com/gfroerli/api/pull/70/commits/21725b23a30f37e555c2b921d8c0afb813a946c8